manageiq-schema.gemspec

```diff
@@ -19,6 +19,7 @@ Gem::Specification.new do |s|
   s.add_dependency "more_core_extensions"
   s.add_dependency "pg", "~> 0.18.2"
   s.add_dependency "rails", "~> 5.0.2"
+  s.add_dependency "pg-pglogical", "~> 2.0.0"
```

Comment:
Why is this needed?

Reply:
It's not! Looks like I got bit by switching branches around ...